docs/documentation/using-notify.md

```diff
@@ -0,0 +1,48 @@
+To set it locally, run this command in your Terminal, in the root
```

Comment:
It feels like there's some context missing here – I don't know what Notify is, and whether I want to (or need to) use it. The guidance is talking about 'setting it' but I have no idea what 'it' is.